core/tracing/tracer.go

```diff
@@ -0,0 +1,78 @@
+package tracing
+
+import (
+	"time"
+
+	"go.uber.org/fx/core/ulog"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/uber-go/tally"
+	jaegerconfig "github.com/uber/jaeger-client-go/config"
+)
+
+// InitGlobalTracer instantiates a new global tracer
+func InitGlobalTracer(
+	cfg *jaegerconfig.Configuration,
```

Comment:
Keeping this consistent with other core components like logging. Check service.go